README.md

```diff
@@ -125,8 +125,10 @@ The configuration provided here is configuration that is shared across component
 ```js
 function initAnswers() {
   ANSWERS.init({
-    // Required, your Yext Answers API key
+    // Required*, your Yext Answers API key. *Do NOT provide apiKey if token is used.
     apiKey: '<API_KEY_HERE>',
+    //Required*, custom auth token. *Do NOT provide token if apiKey is used.
```

Comment:
Space after //. For my edification what do the '*'s do?

Reply:
I following the approach similar to how `businessId` is documented with 'Optional*' below. It indicates that it's required, with an exception if another authentication method is used. So in this case, token is required unless apiKey is used.